src/lib/sedlexing.mli

```diff
@@ -156,8 +162,10 @@ val backtrack: lexbuf -> int
     (the current position is set to the value of the backtrack position). *)
 
 val with_tokenizer: (lexbuf -> 'token) -> lexbuf -> (unit -> 'token * Lexing.position * Lexing.position)
-(** [with_tokenizer tokenizer lexbuf] returns a generator of tokens annotated with positions.
-    This generator can be used with {!TheRightMenhirFunction} *)
+(** [with_tokenizer tokenizer lexbuf] given a lexer and a lexbuf,
+    returns a generator of tokens annotated with positions.
+    This generator can be used with the Menir parser generator's
+    incremental API. *)
```

Comment:
`Menhir`*. It would be nice to give the exact appropriate call here.

Reply:
It's not quite so easy. Pages 27 and 28 of the Menhir manual explain.